projects/valtimo/case-management/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@valtimo/dossier-management",
+  "name": "@valtimo/case-management",
   "license": "EUPL-1.2",
-  "version": "12.2.0",
+  "version": "13.0.0",
```

Comment:
Replace version everywhere (root package.json, libs' package.json) while you're at it